File: dataset/output-bk/MISTRAL_codestral-latest/Task101_MISTRAL_codestral-latest.java
import java.io.File;
import java.io.IOException;

public class Main {
    public static void main(String[] args) {
        try {
            File file = new File("script.sh");
            if (file.createNewFile()) {
                System.out.println("File created: " + file.getName());
                if (file.setExecutable(true)) {
                    System.out.println("File is now executable");
                } else {
                    System.out.println("Failed to make file executable");
                }
            } else {
                System.out.println("File already exists.");
            }
        } catch (IOException e) {
            System.out.println("An error occurred.");
            e.printStackTrace();
        }
    }
}

File: dataset/output-bk/MISTRAL_codestral-latest/Task101_MISTRAL_codestral-latest.c
#include <stdio.h>
#include <stdlib.h>
#include <sys/stat.h>

int main() {
    FILE *file = fopen("script.sh", "w");
    if (file) {
        printf("File created: script.sh
");
        fclose(file);
        if (chmod("script.sh", S_IRWXU) == 0) {
            printf("File is now executable
");
        } else {
            printf("Failed to make file executable
");
        }
    } else {
        printf("Failed to create file
");
    }
    return 0;
}